### DISCUSSION TITLE
multiple selection field not working on Qfield version 3.2.0

### DISCUSSION THREAD

**Discussion by mahmoodi-source** [opengisch/QField#5111] [Q&A]
I have configured the attribute form of my project value relation using a table with no geometry to create the multiple selection field, which is not working in QField version 3.2.0-congo, while it was working in the previous version 2.8.1-insightful perfectly. anyone knows what is the problem?

**Comment by suricactus** [comment]
Preparing a sample project that reproduces the issue always increases your chances for someone to help you with your problem.

**Comment by mahmoodi-source** [comment]
I was using an expression attribute form configuration for the old version of Qfield 2.8.1-Insightful ( "Verk_ph" =current_value('Verkehrsplanung')) to be able to have multiple selections. It turns out that I did not need that expression for the newer version of Qfield (3.2.0-congo). When I removed that expression, it worked perfectly.
Thanks
